### ISSUE TITLE
QuantileLinearRegression causing __init__() got an unexpected keyword argument 'normalize'

### ISSUE THREAD

**Issue by salmazenn** [sdpython/mlinsights#114]
Hi there, 

I have noticed that due to (https://github.com/sdpython/mlinsights/commit/a437ba8578dcc3a872e0faa0a74492dfa27ce285)   (Or at least, I suspect that  it is the cause of the issue I report below  :)  )

The following code is no longer working (I used the same code during the summer, but is no longer working)
```
QuantileLinearRegression(fit_intercept=True,
                         max_iter=50,
                         quantile=0.5,
                         verbose=False)
```
Is this function call deprecated as well?

I am getting this exception()

```
 __init__() got an unexpected keyword argument 'normalize'
```

I also tried to fallback to using `QuantileRegression` from sklearn (As shown below,  failinf to find the parameter to set `max_iter` in sklearn's model), but it doesn't seem to fit the data the same way,  so I would love to continue using mlinsights :)

```
from sklearn.linear_model import QuantileRegressor
quantile_regressor = QuantileRegressor(fit_intercept=True, quantile=0.5)
```  

Thanks!

**Comment by salmazenn** [comment]
Seems to be dependant on library versions,  would be helpful if all library versions are listed on the requirements files

**Comment by xadupre** [comment]
I was fixed in #113.

**Comment by sdpython** [comment]
I'll close this issue. Feel free to reopen it.
